### ISSUE TITLE
Feature enhancement

### ISSUE THREAD

**Issue by DCox2016** [venturedrake/laravel-crm#38]
- Laravel CRM Version: 0.18.1
- PHP Version: 8.1.14

### Description:
Add CSS.php configuration file that will all user to set env variables to overwrite the default CSS, Fonts, and Favicon.

### Steps To Reproduce:
NA

**Comment by DCox2016** [comment]
I cloned this package made my changes and I am unable to push the code. How do I submit this branch to you ?

**Comment by andrewdrake** [comment]
Here is a step by step on how to get to the stage of submitting a pull request - https://www.dataschool.io/how-to-contribute-on-github/

**Comment by DCox2016** [comment]
Hi VentureDrake, team sorry it has be a while. I have been heads down after I found out that I could publish the views. The Limnocity team would like to work with you. We have successfully integrated the the venturedrake_crm into the our  system, but we are finding bugs that we would like to bring to you attention and hopefully help resolve them. As it is stated, the project is not production ready.

**Comment by andrewdrake** [comment]
Yes although I wouldn’t call it production ready it is being used in a few production projects. Publishing the views is ok, keep in mind however that upgrades take a bit more work. In this situation I would only publish views you need to customize. If you republish the entire package views you can overwrite your updates. 

A lot of dev is going on and bugs are being fixed, updated often will help. Follow the upgrade section in the readme. Current version is at 0.19.9

As you find bugs log GitHub tasks and will fix them asap.

I’d you want some help with your send me an email at andrew@venturedrake.com

**Comment by andrewdrake** [comment]
See the recently released version 2, whole new front end stack with Tailwind
